Openafs linux 4.4

RobvanW opened the pull request:
Sorry made an error, wanted just to tryout on my Arch (openafs-modules-dkms doesn't compile with 4.4 kernel)

michaellass committed:
Proposed patches for openafs and linux 4.4

Review:
http://gerrit.openafs.org/12169
http://gerrit.openafs.org/12170

michaellass committed:
New revision of patch 2

michaellass commented:
I am a bit reluctant to distribute this on AUR already. A developer raised concerns that other, more subtle changes in Linux 4.4 may cause data loss with current OpenAFS. Unfortunately I cannot link to this mail, as the mailing list archive is currently broken.

Also, no one but me ever tested these patches and provided feedback. This includes people raising the issue in bug trackers and on mailing lists:
https://rt.central.org/rt/Ticket/Display.html?id=132677
https://lists.openafs.org/pipermail/openafs-devel/2016-January/020289.html
My own testing was very brief as well.

Thorough testing of these patches would be very helpful. But keep the possible data loss issue in mind.

michaellass committed:
Add another patch for Linux 4.4

michaellass committed:
openafs-modules-dkms: No need for install script

Pacman now has a hook to handle DKMS packages properly. Manual adding, building
and removing is not necessary anymore.

Don't bump pkgrel for now as this package is currently broken anyway.

(cherry picked from commit e3890fac27abfb56a04e05691d37d9c23db84c18)

michaellass committed:
New revision of patch 3
